niceml/dlframeworks/tensorflow/callbacks/csvlogger.py

```diff
@@ -43,12 +67,23 @@ def on_epoch_end(self, epoch, logs=None):
 
         row_dict = {"epoch": epoch + 1}
         row_dict.update({k: np.round(v, 6) for k, v in logs.items()})
-        self.data = self.data.append(row_dict, ignore_index=True)
+        self.data = pd.concat([self.data, pd.DataFrame([row_dict])], ignore_index=True)
         self.flush()
 
     def on_train_end(self, logs=None):
+        """
```

Comment:
Missleading docstring, which does not describe what is done but only when it is called. Please rewrite.

Reply:
Done